src/common/cursor.js

```diff
@@ -5,15 +5,12 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 
-import {Effects} from "reflex";
```

Comment:
Removing this import causes flow to fail, as type annotation below (on like 13) refers to `Effects` that is no longer in scope.

I imagine two reasons why web storm is deciding to remove this import:

1. Because it does not actually understands type annotations so assumes that imports are redundant.
2. Because `Effects` is referred from comment web storm ignores it and assumes import is redundant.

I think we could actually use `import type {Effects} from "reflex"` as we only refer to `Effects` from the type position. Which I presume web storm won't remove and ignore. If on the other hand web storm is actually smart enough and No 1 isn't the issue here, then uncommenting return type annotation on line 13 is likely going to be solution for web storm to see that `Effects` is used.

P.S. Fixing this may uncover more type errors due to flow being able to analyze more if it does not fail.